libraries/protocol/include/scorum/protocol/scorum_operations.hpp

```diff
@@ -167,7 +167,7 @@ struct comment_options_operation : public base_operation
     std::string permlink;
 
     asset max_accepted_payout
```

Comment:
I would remove max_accepted_payout. The only perpose it's used is  checking max value in util::get_rshare_reward()

Reply:
This is used for business logic. Look get_rshare_reward